bio/ngscheckmate/ncm/wrapper.py

```
@@ -0,0 +1,128 @@
+            make_list_file_for_fastq(
+                paths=samples,
+                output=list_file,
+                is_paired=snakemake.params.get("paired", True),
+            )
+
+            # Only ncm_fastq.py handles multithreading
+            extra += f" --maxthread {snakemake.threads} "
+        else:
+            raise ValueError("Mixed BAM/VCF/Fastq. Please provide only one file type.")
+
+    list_file = snakemake.input.get("list_file", list_file)
```

Ruff (0.8.0)

Undefined name `snakemake`

(F821)